src/components/slide-toggle/slide-toggle.ts

```diff
@@ -202,13 +212,101 @@ export class MdSlideToggle implements AfterContentInit, ControlValueAccessor {
     }
   }
 
+  /** Emits the change event to the `change` output EventEmitter */
   private _emitChangeEvent() {
     let event = new MdSlideToggleChange();
     event.source = this;
     event.checked = this.checked;
     this._change.emit(event);
   }
 
+
+  /** @internal */

```

Comment:
As of #781 we're avoiding `@internal` everywhere now. Mark it as `TODO: internal` for now.